layout-server/src/test/java/org/finos/vuu/layoutserver/integration/LayoutIntegrationTest.java

```diff
@@ -0,0 +1,389 @@
+package org.finos.vuu.layoutserver.integration;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.hamcrest.Matchers.anyOf;
+import static org.hamcrest.Matchers.equalTo;
+import static org.hamcrest.Matchers.is;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.delete;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.get;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.post;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.put;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.content;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.jsonPath;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.jayway.jsonpath.JsonPath;
+import java.util.UUID;
+import org.finos.vuu.layoutserver.dto.request.LayoutRequestDTO;
+import org.finos.vuu.layoutserver.dto.request.MetadataRequestDTO;
+import org.finos.vuu.layoutserver.model.BaseMetadata;
+import org.finos.vuu.layoutserver.model.Layout;
+import org.finos.vuu.layoutserver.model.Metadata;
+import org.finos.vuu.layoutserver.repository.LayoutRepository;
+import org.finos.vuu.layoutserver.repository.MetadataRepository;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.web.servlet.AutoConfigureMockMvc;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.http.MediaType;
+import org.springframework.test.context.ActiveProfiles;
+import org.springframework.test.web.servlet.MockMvc;
+import org.springframework.test.web.servlet.MvcResult;
+import org.springframework.transaction.annotation.Transactional;
+
+@SpringBootTest
+@AutoConfigureMockMvc
+@Transactional
+@ActiveProfiles("test")
+public class LayoutIntegrationTest {
+
+    private static String defaultDefinition;
+    private static String defaultName;
+    private static String defaultGroup;
+    private static String defaultScreenshot;
+    private static String defaultUser;
+
+    private final ObjectMapper objectMapper = new ObjectMapper();
+
+    @Autowired
+    private MockMvc mockMvc;
+    @Autowired
+    private LayoutRepository layoutRepository;
+    @Autowired
+    private MetadataRepository metadataRepository;
+
+    @BeforeAll
+    public static void setup() {
+        defaultDefinition = "Default layout definition";
+        defaultName = "Default layout name";
+        defaultGroup = "Default layout group";
+        defaultScreenshot = "Default layout screenshot";
+        defaultUser = "Default layout user";
+    }
+
+    @Test
+    void getLayout_validIDAndLayoutExists_returns200WithLayout() throws Exception {
```

Comment:
To be honest I autogenerate the boilerplate of my test files with intellij, and no access modifier is what it defaults to. I don't have a particular opinion on it, although I like [this answer](https://stackoverflow.com/a/55230350) because it involves less typing 😄

> > _Why is the default access modifier in JUnit 5 package-private?_
> 
> It's not the "default". There technically is no default. Rather, in JUnit Jupiter you have a choice: public, protected or package-private.
> 
> > _What is the benefit of changing it to package-private?_
> 
> The benefit is that you don't have type public anymore. If your IDE automatically generates test methods and test classes for you that are public, feel free to leave them public.
> 
> But... if you are typing in the methods on your own, then just leave off public unless you are designing your test classes for subclassing from other packages, in which case you'd want to make your overrideable test methods either public or protected. And of course, interface default methods must be public.
> 
> Long story, short: we (the JUnit 5 team) believe in the principle "Less is more", meaning the less you have to type to achieve your goal, the better!